Userland/Libraries/LibLine/Editor.cpp

```diff
@@ -200,7 +200,7 @@ void Editor::ensure_free_lines_from_origin(size_t count)
 {
     if (count > m_num_lines) {
         // It's hopeless...
-        TODO();
+        VERIFY_NOT_REACHED();
```

Comment:
Please don't do this, `VERIFY_NOT_REACHED` is for situations where a condition is logically or practically impossible, or is otherwise not possible to handle.
This scenario is certainly possible to handle, just that the way to handle it is...not really obvious (at least to me)

Reply:
I thought that setting the terminal to a small size was the only way to this situation.
And as I had a "solution" to this, I thought it was relevant.